vignettes/netCDF_Functionality.Rmd

```diff
@@ -0,0 +1,400 @@
+---
+title: "Demonstration of netCDF functionality in rSW2analysis"
+author: "Caitlin M. Andrews"
+date: "`r Sys.Date()`"
+output:
+  rmarkdown::html_vignette:
+    toc: true
+    toc_depth: 2
+vignette: >
+  %\VignetteIndexEntry{Demonstration of netCDF functionality in rSW2analysis}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+# Overview
+
+This vignette is an overview of how to use the functions in the rSW2analysis package to
+convert arrays of rSOILWAT2 and SOILWAT2 output data into netCDFs that adhere to
+[CF conventions](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.8/cf-conventions.html#_orthogonal_multidimensional_array_representation).
+
+Additionally, this document describes the Dryland Ecology Lab's standards and
+conventions in regards to naming, metadata, and netCDF structure.
+
+# Input Data Format and Structure
+
+## Variable Data
+
+The netCDF functions in this package expect an array, in which each row represents
+data for a site. Columns represent one of three things:
```

Comment:
data for a site --> data for a site or gridcell

Reply:
fixed